src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private StorageDao storageDao;
+
+    @BeforeEach
+    void setup() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl(storageDao);
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User validUser = new User("validLogin", "validPassword", 18);
+        registrationService.register(validUser);
```

Comment:
here you tested only one thing - no exception is thrown upon valid user registration. It's hardly enough. If the method is empty and void, it will pass the test :) 
![image](https://github.com/mate-academy/jv-registration-validation/assets/49767049/9693ed7b-8964-4b1b-8694-9e757095b2c8)